### DISCUSSION TITLE
Introducing GitLens+

### DISCUSSION THREAD

**Discussion by eamodio** [gitkraken/vscode-gitlens#2170] [Announcements]
# What is GitLens+

GitLens+ adds all-new, completely optional, features that enhance your current GitLens experience when you sign in with a free account. A free GitLens+ account gives you access to [GitLens+ features](https://www.gitkraken.com/gitlens/plus-features) on local and public repos, while a paid account allows you to use them on private repos.

[GitLens+ features](https://www.gitkraken.com/gitlens/plus-features) include the Commit Graph, Visual File History, Worktrees, and more.

<p align="center">
  <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/commit-graph-illustrated.png" alt="Commit Graph illustration"/>
  <br/>New Commit Graph
</p>

<p align="center">
  <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/visual-file-history-illustrated.png" alt="Visual File History illustration"/>
  <br/>Visual File History
</p>

<p align="center">
  <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/worktrees-illustrated.png" alt="Worktrees illustration"/>
  <br/>Worktrees
</p>

# Why GitLens+

Our intention with GitLens+ is to create a sustainable model that allows us to continue to heavily invest in the free and open source core of GitLens–features that enable greater productivity, safety, ease-of-use, and more–while also creating a set of additional, but completely optional, GitLens+ features designed specifically for teams. 

The GitLens+ features are intended to improve collaboration and visualization, and are still completely free with an account, for use with local and public repositories. These features only become paid when working with private repos. 

You can read more about the state and future of GitLens here: https://www.gitkraken.com/blog/gitlens-gitkraken

**Comment by theklr** [comment]
Is there anyway to disable this? I just want GitLens and would like not to get prompted so much on a work computer.

**Reply by eamodio** [reply]
@theklr You can run the "GitLens+: Hide GitLens+ Features" command or toggle the `gitlens.plusFeatures.enabled` setting directly (note, the command toggles the setting for you). Hope that helps!

**Comment by Alokrar** [comment]
Hi

**Comment by Alokrar** [comment]
Hello
